src/test/python/pythonparser/test_pythonparser_3.py

```diff
@@ -0,0 +1,168 @@
+# Copyright (c) Aniskov N.
+
+import os
+from xml.etree import ElementTree as ET
+from typing import *
+
+import pytest
+
+from src.main.python.pythonparser.pythonparser_3 import *
+from src.main.util.const import RESOURCE_FOLDER_PATH
+
+
+class TestPythonParser3:
+
+    _input_dir = os.path.join(RESOURCE_FOLDER_PATH, 'pythonparser/pythonparser_3')
+
+    test_cases_dirs = \
+        [
+            os.path.join(_input_dir, 'case_small_code_snippets'),
+
+            os.path.join(_input_dir, 'case_big_code_snippets'),
+
+            os.path.join(_input_dir, 'case_classes'),
+
+            os.path.join(_input_dir, 'case_empty_file'),
+
+            pytest.param(os.path.join(_input_dir, 'case_fstrings'),
+                         marks=pytest.mark.skip(reason='fstrings are not supported yet')),
+
+            pytest.param(os.path.join(_input_dir, 'case_python_3_8'),
+                         marks=pytest.mark.skip(reason='python 3.8 not supported yet'))
+        ]
+
+    # Helper functions
+
+    @staticmethod
+    def get_xml_tree(path_to_py_file: str) -> ET.Element:
+        parsed_json = parse_file(path_to_py_file)
+        xml_str = json2xml(parsed_json)
+        root = ET.fromstring(xml_str)
+        return root
+
+    @staticmethod
+    def count_token_occurrence(tree: ET.Element, tokens: Iterable[str]) -> Dict[str, int]:
```

Comment:
Maybe should we use the verbs in the begin if such kind of functions names for a uniform style? For example, count_token_occurrence -> get_count_of_token_occurrences

It is a public method? In Python there are not the  access modifiers like public, private, but you can use the double underscores to mark it as a private function: `__get_count_of_token_occurrences`. Please use it for all functions which should be private, for example for the `get_xml_tree` function.